components/odh-notebook-controller/controllers/notebook_controller.go

```diff
@@ -184,6 +189,14 @@ func (r *OpenshiftNotebookReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		return ctrl.Result{}, err
 	}
 
+	// Call the Rolebinding reconciler
+	if strings.ToLower(strings.TrimSpace(os.Getenv("SET_PIPELINE_RBAC"))) == "true" {
+		err = r.ReconcileRoleBindings(notebook, ctx)
+		if err != nil {
+			return ctrl.Result{}, err
```

Comment:
Just asking - is this error ever be logged in some clean/readable way? On line 189, I can see that we don't log anything there, but another lines above that e.g. on line 161, we actually log error in a similar situation.

Reply:
done